coordinator/tsconfig.json

```diff
@@ -15,18 +15,16 @@
     "baseUrl": ".",
     "paths": {
       "*": [
-        "node_modules/*",
-        "tests/*"
+        "node_modules/*"
       ],
-      "@models": ["../models", "./models"]
+      "@models": ["../models"]
     }
   },
   "include": [
-    "src/**/*",
-    "tests/**/*"
```

Comment:
We added tests to this file for a reason - this allows the IDE to know that our tsconfig applies to our tests, and so it doesn't complain about things like `flatMap` not existing on arrays. Only the IDE uses this file. The build uses the other tsconfig file which excludes tests